internal/quaiapi/quai_api.go

```diff
@@ -1294,9 +1303,6 @@ func (s *PublicBlockChainQuaiAPI) marshalAuxPowTemplate(wo *types.WorkObject, re
 		targetHex = targetHex[2:]
 	}
 
-	// set the pendingheader time to empty
-	woCopy := types.CopyWorkObjectHeader(wo.WorkObjectHeader())
```

Comment:
Setting time to zero makes it so that sealhash doesnt change just because of the change in time right? i thought this was important for updates